bedrock/mozorg/templates/mozorg/home/includes/structured-data.html

```diff
@@ -0,0 +1,121 @@
+        "sameAs": ["https://en.wikipedia.org/wiki/Mozilla", "https://www.wikidata.org/wiki/Q9661"]
+      },
+      {
+        "@type": "brand",
+        "@id": "https://www.mozilla.org/#firefoxbrand",
+        "url": "https://www.mozilla.org/firefox/",
+        "logo": "{% filter absolute_url %}{{ static('img/logos/firefox/logo-master-wordmark-dark.svg') }}{% endfilter %}",
+        "name": "Firefox",
+        "sameAs": [
+          "https://twitter.com/firefox",
+          "https://www.instagram.com/firefox/",
+          "https://www.youtube.com/firefox"
+        ]
+      },
+      {
+        "@type": "SoftwareApplication",
+        "@id": "https://www.mozilla.org/#firefoxbrowser",
+        "url": "https://www.mozilla.org/firefox/",
+        "image": "{% filter absolute_url %}{{ static('img/logos/firefox/logo-quantum-high-res.png') }}{% endfilter %}",
+        "name": "Firefox Browser",
+        "OperatingSystem": [
+          "MacOS",
+          "Windows",
+          "Linux",
+          "iOS",
+          "Android"
+        ],
+        "offers": {
+          "@type": "Offer",
+          "price": "0.00",
+          "priceCurrency": "USD"
+        },
+        "applicationCategory": "Browser",
+        "sameAs": [
+          "https://en.wikipedia.org/wiki/Firefox",
+          "https://www.wikidata.org/wiki/Q698"
+        ]
+      },
+      {
+        "@type": "Product",
+        "@id": "https://www.mozilla.org/#firefoxmonitor",
+        "url": "https://monitor.firefox.com",
+        "logo": "{% filter absolute_url %}{{ static('img/logos/firefox/logo-monitor.svg') }}{% endfilter %}",
+        "name": "Firefox Monitor",
+        "sameAs": "https://www.wikidata.org/wiki/Q64828890"
+      },
+      {
+        "@type": "Product",
+        "@id": "https://www.mozilla.org/#firefoxlockwise",
+        "url": "https://lockwise.firefox.com",
+        "logo": "{% filter absolute_url %}{{ static('img/logos/firefox/logo-lockwise.svg') }}{% endfilter %}",
+        "name": "Firefox Lockwise",
+        "sameAs": [
+          "https://www.wikidata.org/wiki/Q64829394",
+          "https://en.wikipedia.org/wiki/Firefox_Lockwise"
+        ]
+      },
+      {
+        "@type": "Product",
+        "@id": "https://www.mozilla.org/#firefoxsend",
+        "url": "https://send.firefox.com",
+        "logo": "{% filter absolute_url %}{{ static('img/logos/firefox/logo-send.svg') }}{% endfilter %}",
+        "name": "Firefox Send",
+        "sameAs": [
+          "https://en.wikipedia.org/wiki/Firefox_Send",
+          "https://www.wikidata.org/wiki/Q62394165"
+        ]
+      }
+    ],
+    "parentOrganization": {
+      "@type": "NGO",
+      "@id": "https://foundation.mozilla.org/#organisation",
+      "name": "Mozilla Foundation",
+      "url": "https://foundation.mozilla.org",
+      "sameAs": [
+        "https://www.wikidata.org/wiki/Q55672",
+        "https://foundation.mozilla.org",
+        "https://en.wikipedia.org/wiki/Mozilla_Foundation"
+      ]
+    }
+  }]
```

Comment:
I believe the json here is invalid because the `}]` here doesn't match up to anything. Removing it seems to fix things (this is pretty tricky to work out when it's inline in HTML like this, something to think about for the future).

Reply:
Nice catch, I organized this json first as a graph but at the end decided against it with removing website and webpage, these have been the end of the graph.